crates/sui-framework/packages/bridge/sources/chain_ids.move

```diff
@@ -38,18 +46,42 @@ module bridge::chain_ids {
         EthSepolia
     }
 
+    public fun eth_local_test(): u8 {
+        EthLocalTest
+    }
+
     public fun valid_routes(): vector<BridgeRoute> {
         vector[
             BridgeRoute { source: SuiMainnet, destination: EthMainnet },
+            BridgeRoute { source: EthMainnet, destination: SuiMainnet },
+
             BridgeRoute { source: SuiDevnet, destination: EthSepolia },
+            BridgeRoute { source: SuiDevnet, destination: EthLocalTest },
             BridgeRoute { source: SuiTestnet, destination: EthSepolia },
-            BridgeRoute { source: EthMainnet, destination: SuiMainnet },
+            BridgeRoute { source: SuiTestnet, destination: EthLocalTest },
+            BridgeRoute { source: SuiLocalTest, destination: EthLocalTest },
+            BridgeRoute { source: SuiLocalTest, destination: EthSepolia },
             BridgeRoute { source: EthSepolia, destination: SuiDevnet },
-            BridgeRoute { source: EthSepolia, destination: SuiTestnet }]
+            BridgeRoute { source: EthSepolia, destination: SuiTestnet },
+            BridgeRoute { source: EthSepolia, destination: SuiLocalTest },
+            BridgeRoute { source: EthLocalTest, destination: SuiDevnet },
+            BridgeRoute { source: EthLocalTest, destination: SuiTestnet },
+            BridgeRoute { source: EthLocalTest, destination: SuiLocalTest }
+        ]
     }
 
     public fun is_valid_route(source: u8, destination: u8): bool {
         let route = BridgeRoute { source, destination };
         return vector::contains(&valid_routes(), &route)
     }
+
+    // Checks and return BridgeRoute if the route is supported by the bridge.
+    public fun get_route(source: u8, destination: u8): BridgeRoute {
+        let route = BridgeRoute { source, destination };
+        return if (vector::contains(&valid_routes(), &route)) {
+            route
+        } else {
+            abort EInvalidBridgeRoute
```

Comment:
nit: we should prefer assertions instead of aborts in single conditional scenarios.